class.jetpack-gutenberg.php

```diff
@@ -689,4 +689,46 @@ public static function load_independent_blocks() {
 			}
 		}
 	}
+
+	/**
+	 * Get CSS classes for a block.
+	 *
+	 * @since 7.7.0
+	 *
+	 * @param string $slug  Block slug.
+	 * @param array  $attr  Array containing the block attributes.
```

Comment:
Nitpick -- I typically omit the data type (`Array of`) from the descriptions, since it's already specified more 'formally' next to `@param`. Totally up to you, though.

```suggestion
	 * @param array  $attr  Block attributes.
```

Reply:
I like that, that makes perfect sense. I'll try to remember that from now on.